language/move-vm/natives/src/debug.rs

```diff
@@ -25,12 +25,11 @@ pub fn native_print(
     // No-op if the feature flag is not present.
     #[cfg(feature = "debug_module")]
```

Comment:
out of curiosity, do we compile with the feature (`debug_module`) on? off? when?
I know we talked about this but I am not sure I remember well right now

Reply:
AFAIK we only turn this feature on with `e2e-tests`.